frontend/src/Components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Nav } from "react-bootstrap";
+import "./Navbar.css";
+import Sidebar from "../Sidebar/Sidebar";
+import Profile from "../Profile/Profile";
+import logoImage from "..//Login/assets/logoblackandwhite.svg";
+
+function Navbar() {
```

Comment:
No big deal, but I noticed some components are arrow functions, some functions. I would stick to one style.